README.md

```diff
@@ -27,3 +27,48 @@ We want to provide a single input interface to the system (pipeline index) and d
     * Probably part of Input Interface, presenting logs from failed builds
 
 
+## Usage
+
+### Vagrant
+
+### Setup environment
+
+#### CentOS
+```
+# Install dependencies
+sudo yum install -y epel-release git
+sudo yum install -y ansible1.9 centos-release-scl qemu-kvm libvirt sclo-vagrant1
+
+# start libvirtd
+sudo systemctl start libvirtd; sudo systemctl enable libvirtd
+
+# enable bash on vagrant scl
+sudo scl enable sclo-vagrant1 bash
+```
+
+#### Fedora
+```
+# sudo dnf install -y git ansible vagrant
+```
+
+### Get the code and install vagrant plugins
+
+```
+git clone https://github.com/CentOS/container-pipeline-service
+cd container-pipeline-service
+vagrant plugin install vagrant-host-manager
```

Comment:
In PR: #11, I have removed dependency on ``vagrant-host-manager`` plugin, so I will update the docs accordingly.